### ISSUE TITLE
C extension modules

### ISSUE THREAD

**Issue by kramasamy** [google/subpar#59]
Wondering if there is any roadmap/ETA for supporting C extension modules.

**Comment by duggelz** [comment]
I don't have a specific ETA, sorry, but it is something we're working on, and trying to staff up.

**Comment by hwright** [comment]
@duggelz Are those design discussions and planning happening some place public?  This is relevant to my interests, and in the spirit of open source I would mind contributing if I knew where the needs were.

**Comment by duggelz** [comment]
Yes, those design discussions will be public once I have something to share.  RIght now, they are pretty much all just happening inside my head.  We are currently doing internal staffing and product prioritization discussions, since this started as a 20% project, which is clearly inadequate.

**Comment by c4urself** [comment]
Any update on this @duggelz?

**Comment by nicolov** [comment]
+1, it would be nice to have updates about this. From what I could gather, it seems that Google has a patched libc to allow `dlopen` from within a zip file, right?

**Comment by blais** [comment]
+1 I need support for C extension modules as well...

**Comment by animesh-sharama** [comment]
+1 I also need support for C extension.
Can't use this with grpc since grpc relies on cython ```ImportError: cannot import name 'cygrpc' from 'grpc._cython' ```
